en/releases/1.14.md

```diff
@@ -1,4 +1,17 @@
-## PX4-Autopilot v1.14 Release Notes
+# PX4-Autopilot v1.14 Release Notes
+
+## Upgrading to PX4 v1.14
+
+PX4 v1.14 contains many new features, feature upgrades, and bug fixes.
+This section outlines the most important changes to consider when upgrading: those that could affect the stability of your vehicle, and their integration with other systems.
+
+1. When updating you should use the [Actuator Configuration UI](../config/actuators.md) to confirm that the geometry of the airframe matches your vehicle, and that the correct functions (motors and servos) are mapped to outputs as they are wired to the frame and with the correct ESC type specified: the test sliders in the UI an be used to confirm motor wiring.
+
+   If using PWM ESC for motors then running an [ESC Calibration](../advanced_config/esc_calibration.md) is highly recommended, and then setting appropriate disarmed, minimum and maximum values for the motors (in the actuator UI).
+   
+   This is critical if you are using a custom mixer file or the airframe you assigned in an earlier version is not present in PX4 v1.14.
+   However it is still recommended even if you are using an airframe that precisely matches a specific vehicle in the [Airframe Reference](../airframes/airframe_reference.md) (such as the [Holybro X500 V2](#copter_quadrotor_x_holybro_x500_v2)) as your wiring and ESCs calibration may not match the defaults.
+1. Default disarmed PWM was changed from 900us to 1000us. So make sure to check whether you were using default PWM disarmed values, and if it works with your setup. You can find related information in the [step 7 of ESC calibration](../advanced_config/esc_calibration.md#steps) document for details.
 
```

Comment:
Add pointer to the ROS 2 migration guidelines

```suggestion
1. ROS 2/DDS users will need to modify their code to use uXRCE-DDS (instead of Fast-RTPS, which it replaces in PX4 v1.14).
   For more information see [uXRCE-DDS > Fast-RTPS to uXRCE-DDS Migration Guidelines](../middleware/uxrce_dds.md#fast-rtps-to-uxrce-dds-migration-guidelines)

```